client/src/app/queries/assessments.ts

```diff
@@ -224,40 +223,39 @@ const removeSectionOrderFromQuestions = (
 export const useFetchAssessmentsWithArchetypeApplications = () => {
   const { assessments, isFetching: assessmentsLoading } = useFetchAssessments();
 
+  const archetypeQueries = assessments
+    .map((assessment) => assessment?.archetype?.id)
+    .filter(Boolean)
+    .map((archetypeId) => ({
```

Comment:
Are assessments a 1:1 relationship to archetypes?  If multiple assessments can be linked to the same archetype then this list could contain the same archetype id multiple times and have multiple requests for that archetype.  Probably wouldn't be a big issue, but can be avoided.